common/config/rush/artifactory.json

```diff
@@ -0,0 +1,85 @@
+/**
+ * This configuration file manages Rush integration with JFrog Artifactory services.
```

Comment:
Was this file intentionally added?

Reply:
Yes, it's part of the standard `rush init` template.